app/views/hyku/api/v1/work/_work.json.jbuilder

```diff
@@ -23,10 +23,12 @@ json.cache! [@account, :works, work.id, work.solr_document[:_version_], work.mem
 
   creator_hash = work.creator.try(:first)
   creator_json = JSON.parse(creator_hash) if creator_hash.present?
-  creator_json.each_with_index do |creator, index|
-    creator_json[index] = creator_json[index].except!("creator_institutional_email") && next unless creator["creator_institutional_email"].present? # Removes field if blank
-    user = User.find_by(email: creator["creator_institutional_email"])
-    creator_json[index] = creator_json[index].except!("creator_institutional_email") unless user.present? && user.display_profile # Removes field if the user is not public/not found
+  if creator_json.present?
+    creator_json.each_with_index do |creator, index|
+      creator_json[index] = creator_json[index].except!("creator_institutional_email") && next unless creator["creator_institutional_email"].present? # Removes field if blank
```

Comment:
You're doing too much on one line here, assigning a variable, moving to the next iteration and checking present. Its ok to use more than one line if you are going multiple things and it makes the code easier to read. A giant block of code needs to be spaced to that the next developer that comes to it can process it quickly. 

Also, avoid `unless present?` if you can use `if blank?`. 

```
if creator["creator_institutional_email"].blank?
    creator_json[index] = creator_json[index].except!("creator_institutional_email") 
    next 
end
```